Revert https://github.com/any1/neatvnc/commit/a43bb5d3d438 in favor of CPUTYPE.
amd64 implies SSE2 while not all armv6 have NEON.

--- meson.build.orig	2020-07-11 20:16:27 UTC
+++ meson.build
@@ -21,14 +21,6 @@ if buildtype != 'debug' and buildtype != 'debugoptimiz
 	c_args += '-DNDEBUG'
 endif
 
-cpu = host_machine.cpu_family()
-
-if cpu == 'x86_64'
-	c_args += '-m' + get_option('x86_64-simd')
-elif cpu == 'arm'
-	c_args += '-mfpu=neon'
-endif
-
 add_project_arguments(c_args, language: 'c')
 
 cc = meson.get_compiler('c')
